openquake/smt/comparison/utils_gmpes.py

```diff
@@ -24,43 +24,13 @@
 from openquake.hazardlib.geo import Point
 from openquake.hazardlib.geo.surface import PlanarSurface
 from openquake.hazardlib.source.rupture import BaseRupture
-from openquake.hazardlib.geo import utils as geo_utils
 from openquake.hazardlib.geo.geodetic import npoints_towards
 from openquake.hazardlib.site import Site, SiteCollection
 from openquake.hazardlib.scalerel import WC1994
 from openquake.hazardlib.const import TRT
 from openquake.hazardlib.contexts import ContextMaker
 from openquake.hazardlib.gsim.mgmpe import modifiable_gmpe as mgmpe
-
-
-def _get_first_point(rup, from_point):
-    """
-    :param rup:
-    :param from_point:
-    """
-    sfc = rup.surface
-    if from_point == 'TC':  # Get the up-dip edge centre point
-        return sfc._get_top_edge_centroid()
-    elif from_point == 'BC':  # Get the down-dip edge centre point
-        lon, lat = geo_utils.get_middle_point(
-            sfc.corner_lons[2], sfc.corner_lats[2],
-            sfc.corner_lons[3], sfc.corner_lats[3]
-        )
-        return Point(lon, lat, sfc.corner_depths[2])
-    elif from_point == 'TL':
-        idx = 0
-    elif from_point == 'TR':
-        idx = 1
-    elif from_point == 'BR':
-        idx = 2
-    elif from_point == 'BL':
-        idx = 3
-    else:
-        raise ValueError('Unsupported option from first point')
-    return Point(sfc.corner_lons[idx],
-                 sfc.corner_lats[idx],
-                 sfc.corner_depths[idx])
-
+    
 
 def get_sites_from_rupture(rup, from_point='TC', toward_azimuth=90,
```

Comment:
If you remove the `_get_first_point` function, this param is no longer required.